apps/prairielearn/src/pages/shared/syncHelpers.ts

```diff
@@ -9,17 +7,22 @@ import {
 import * as async from 'async';
 import Docker from 'dockerode';
 
-import { DockerName, setupDockerAuth } from '@prairielearn/docker-utils';
+import { type DockerAuth, DockerName, setupDockerAuth } from '@prairielearn/docker-utils';
 import * as Sentry from '@prairielearn/sentry';
 
 import { makeAwsClientConfig } from '../../lib/aws.js';
 import { config } from '../../lib/config.js';
 import { pullAndUpdateCourse } from '../../lib/course.js';
-import { createServerJob } from '../../lib/server-jobs.js';
+import { createServerJob, type ServerJob } from '../../lib/server-jobs.js';
 
 const docker = new Docker();
 
-export async function pullAndUpdate(locals) {
+/**
+ *
+ * @param locals res.locals
+ * @returns jobSequenceId
```

Comment:
```suggestion
 * @returns The ID of the job sequence created for this process
```
Similar to other cases below.